oqamaint/ReviewBot.py

```diff
@@ -0,0 +1 @@
+../ReviewBot.py
```

Comment:
Is this considered good practice in python world? In other languages one would use from parent location.

It seems relative imports to parent directory is also a thing in python.

Reply:
no relative imports work only inside of the package. So not usable in this case.

Reply:
Perhaps you could spend the effort to setup `osclib` as a python module and fix spec so it could be loaded properly as the current layout of repo is a bit cumbersome and most things seem to be in root directory to avoid this issue.

Reply:
Perhaps. .. osclib maybe , How about ReviewBot.py ?

Reply:
Probably wishful thinking since apparently it only works on modules. Not sure why there is not a better solution to this. Works fine in other languages.

Reply:
I don't know what they were required for - the bot works fine without it.